package.json

```diff
@@ -2,7 +2,7 @@
   "name": "quoting-service",
   "description": "Quoting Service hosted by a scheme",
   "license": "Apache-2.0",
-  "version": "14.0.0",
+  "version": "14.0.1",
```

Comment:
Don't bump the version as it will be handled automatically by the CI Release process.

Reply:
done, ready for final review